BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift

```diff
@@ -65,15 +80,15 @@ extension ComScoreBitmovinAdapter: PlayerListener {
     }
 
     func onPaused(_ event: PausedEvent) {
-        stop()
+        // ComScore only wants us to call stop if we are NOT in an ad break
+        if !player.isAd {
+            stop()
+        }
     }
 
     func onPlay(_ event: PlayEvent) {
-        if player.isAd {
-            playAdContentPart(duration: currentAdDuration, timeOffset: currentAdOffset)
-        } else {
-            playVideoContentPart()
-        }
+        // ComScore only wants us to resume into video content. We should not transition state when pause / play is called in an ad
+        resume()
```

Comment:
shouldn't there be also the `if !player.isAd {` check to prevent resume when pressing pause / play during an add?